### PR TITLE
[SDK-259] change meshlod to lod

### PR DESCRIPTION
**Pull request by rk132**
## Dependencies
```Package
Avatar Loader: branch
```

## Description

-   pass lod parameter to the avatar loader query instead of meshlod

#### Updated

-   pass lod parameter to the avatar loader query instead of meshlod

## How to Test

- Open up test scene for avatar lod
- When changing distance, the LOD is also applied to the head.

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[SDK-259]: https://ready-player-me.atlassian.net/browse/SDK-259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Tests/Editor/AvatarLoaderTests.cs [hunk 1]
```diff
@@ -11,7 +13,8 @@ namespace ReadyPlayerMe.Core.Tests
     public class AvatarLoaderTests
     {
         private GameObject avatar;
-        
+        private GameObject avatar2;
```
Comment (HarrisonHough) [inline Tests/Editor/AvatarLoaderTests.cs:16]: This is actually not necessary, you can just re-use the same avatar object field, overriding it once you have the vertex count of the first loaded avatar.

**Review by HarrisonHough — CHANGES_REQUESTED** [review]
Looks good! Just 1 small suggestion in the comments to simplify things a little.

File: Tests/Editor/AvatarLoaderTests.cs [hunk 2]
```diff
@@ -173,5 +180,37 @@ public IEnumerator AvatarLoader_Cancel_Loading()
             Assert.AreNotEqual(FailureType.None, failureType);
             Assert.AreEqual(null, avatar);
         }
+
+        [UnityTest, CanBeNull]
```
Comment (HarrisonHough) [inline Tests/Editor/AvatarLoaderTests.cs:184]: Also forgot to mention, why was this CanBeNull macro added? 

I believe it's not necessary, and can be removed along with the 
```cs
using JetBrains.Annotations;
```
At the top

**Review by rYuuk — COMMENTED** [review]
Please can you align the title of PR with our PR formats(you can check the closed PRs). Also for future, regarding branch name, we don't use ticket number in there as it feels redundant.

**Review by HarrisonHough — APPROVED** [review]
Looks good. 
Great stuff!

**Pull request merged**